app/Models/Permission.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\SoftDeletes;
+
+class Permission extends Model
+{
+    use SoftDeletes;
+    /**
+     * [$table description]
+     * @var string
+     */
+    protected $table="permissions";
+    /**
+     * [$dates description]
+     * @var array
+     */
+    protected $dates = ['deleted_at'];
+    /**
+     * [$fillable description]
+     * @var [type]
+     */
+    protected $fillable = [
+        'name',
+        'display_name',
+        'description',
+    ];
+    /**
+     * Get roles
+     * @return [type] [description]
+     */
+    public function roles() {
+        return $this->belongsToMany('App\Models\Role', 'permission_role', 'permission_id', 'role_id');
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.BraceOnSameLine</br>**Severity**: error</br>**File**: app/Models/Permission.php</br>Opening brace should be on a new line